Flask_ORM/many_to_many/routes.py

```diff
@@ -0,0 +1,55 @@
+from flask import request
+from flask_restful import Resource, marshal_with, reqparse
+import json
+from db import db
+from many_to_many.structure import staff_structure, room_structure
+
+from models.models import StaffModel, RoomsModel
+
+
+staff_room_parser = reqparse.RequestParser()
+staff_room_parser.add_argument("name", required=True, help="Important: {error_msg}!!!")
+
+
+class StaffView(Resource):
+    @marshal_with(staff_structure)
+    def get(self):
+        return StaffModel.query.all()
+
+    def post(self):
+        data = json.loads(request.data)
+        new_staff = StaffModel(**data)
+        db.session.add(new_staff)
+        db.session.commit()
```

Comment:
what if data were not saved? catch possible errors

Reply:
done